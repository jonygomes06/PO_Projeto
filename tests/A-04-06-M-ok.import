BOOK:Bairro Alto e o Budismo Zen:Zun Tse Fonseca:25:FICTION:1234567891:50
BOOK:Analise Matematica sem Mestre:Jorge Espiga, Carlos Fonseca:29:SCITECH:1234567890:5
DVD:Era uma vez na Amadora:Fernando Fonseca:20:FICTION:200505550:10
DVD:48 Horas para o Exame:Orlando Fonseca:12:FICTION:200505553:10
BOOK:A arte de sobreviver no 36:Joao Fonseca, Brázio Rodrigues, Ana Dias, Carlos Amaral, António:30:FICTION:1234567892:22
BOOK:Dicionário de Programação:Odete Fonseca:20:REFERENCE:1234567890:50
DVD:Lumiar Selvagem:Pedro Fonseca:20:FICTION:200505551:5
