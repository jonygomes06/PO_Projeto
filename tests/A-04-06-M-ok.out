Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o número da obra: 2 - 5 de 5 - Livro - Analise Matematica sem Mestre - 29 - Técnica e Científica - Jorge Espiga; Carlos Fonseca - 1234567890
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o número da obra: 5 - 22 de 22 - Livro - A arte de sobreviver no 36 - 30 - Ficção - Joao Fonseca; Brázio Rodrigues; Ana Dias; Carlos Amaral; António - 1234567892
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 